frame/encode_test.go

```diff
@@ -18,3 +19,61 @@ func (s *EncodeSuite) TestUnencodeValue(c *C) {
 	c.Check(err, IsNil)
 	c.Check(val, Equals, "Contains\r\nNewLine and : colon and \\ backslash")
 }
+
```

Comment:
the `*Old` benchmarks don't really test our code, I understand the motivation but I think it's better to remove them, also the top comment, ideally we can keep those as a comment on the PR please.

Reply:
ok, i remove it.